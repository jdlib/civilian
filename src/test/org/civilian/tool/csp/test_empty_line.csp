package test

template(String name) extends SuperTemplate(name)
{{
	<%%>
}}