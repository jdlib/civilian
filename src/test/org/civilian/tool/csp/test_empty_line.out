/**
 * Generated from test_empty_line.csp
 * Do not edit.
 */
package test;


public class Test_empty_line extends SuperTemplate
{
	public Test_empty_line(String name)
	{
		super(name);
		this.name = name;
	}


	@Override protected void print() throws Exception
	{
		out.println();
		if (true)                                                       // line 6: @if (true)
		{
			out.println();
			out.println("Hello");                                       // line 8: Hello
		}
	}


	protected String name;
}
